source/_posts/2017/2017-01-20-4-emotional-reasons-I-quit-my-twitter.md

```diff
@@ -0,0 +1,128 @@
+ 
+It's like finding passion in relations ship with woman I'm not attracted.
+
+
+What I like about Facebook, Github or PHPStorm is that **they evolve and innovate**. Sometimes I like it, sometime not, but **I know they are trying to be better self**.
+  
+I Twitter would be person, it would be fearful virgin in his 30ties sticking to first job with no girlfriend.  
+
+ 
+
+## 4. I already Love Somebody Else
+
+140 limit for Tweet is fine, but I want to get deeper. I also don't like random Tweet stream: one day 50 tweets, other day 5. What the hell, man? That's so schizophrenic.
+
+**I love system, regularity and predictability**. This the best way to educate constantly in certain speed.
+ 
+That's why I switched from Tweetdeck-hashtag-group-following based to regular newsletter one in last year.
+
+I started to follow: 
+
+- [PHP Annotated Monthly](https://info.jetbrains.com/PHP-Annotated-Subscription.html) from JetBrains (PHPStorm guys)
+- [PHP Weekly](http://www.phpweekly.com/)
+- [Měsíčník pro Péhápkaře](pehapkari.cz/#newsletter)
+- [Web Development Reading List](https://wdrl.info/) (thanks [Jakub Kratina](http://jakubkratina.cz/) for the tip 
+- [The Open Circle](http://marketmeditations.com/theopencircle/)
+- [Jan Růžička - Týden v Médiích](https://www.respekt.cz/echo/neco-konci-neco-zacina)
+- [Eliška Vyhnánková](http://vyhnankova.cz/)
+
+And it's much better.
+
+
+## I Love Experiments
+
+Leo Babauta happily dropped Twitter in 2013. In [Simplify the internet](Simplify the Internet) he wrotes
+
+> If you’re going to do Facebook, don’t do other ones too. You can quit Twitter and Instagram. Really you can!
+
+Reading this first time in December, I felt it's the right thing to do, yet brain opposing with "that insane!".
+
+My great inspiration from polyphasic sleep days Steve Pavlina [got social media dumped hard](http://www.stevepavlina.com/blog/2014/07/social-media-you-got-dumped/).
+
+> I basically asked myself which scenario seemed best over the next 10 years — going social media-free vs. continuing to use it.
+
+Followed by [a year later reflection](http://www.stevepavlina.com/blog/2015/07/one-year-without-social-media/).
+
+I love social experiments. After polyphasic sleep, switching main job focus after 8 years, having a baby while being infertile, dropping from university, moving to Prague after settling down in Brno, [droping LinkedIn](todo linkg) and moving completely alone for first time, I really look forward to this one.
+
+
+## Where do I find you then?
+
+As you can see I love newsletters. I'll be starting my soon and I'm still on Facebook.
+Both you can find on my website [tomasvotruba.cz](https://www.tomasvotruba.cz/#socials).
+
+
+## What about Your Socials Experiments?
+
+I've already asked, why and what social networks do you use. 
+
+Now, I would like to know, **have you made any social network experiment?**
+How did it go? Did you liked that?
+
+Look forward to your answers!
```

Comment:
I googled this: http://dictionary.cambridge.org/grammar/british-grammar/verb-patterns/look-forward-to

I guess is the case in the end of the letter here. So **look foward**...